code/__HELPERS/text.dm

```diff
@@ -297,6 +323,27 @@
 		return message
 	return copytext(message, 1, length + 1)
 
+/*
+ * Misc
+ */
+
+/proc/stringsplit(txt, character)
+	var/cur_text = txt
+	var/last_found = 1

```

Comment:
string split and dd split text were removed because byond created a native function that does that.

did you need to do it this way for a reason or could those case be converted to use `splittext()`